docker-compose.yml

```diff
@@ -1,6 +1,10 @@
-dev:
-  image: haskell:7.10.3
-  volumes:
-    - .:/kioku
-  working_dir: /kioku
+version: '3'
+services:
+  dev:
+    image: flipstone/stack:v2-1.9.3
+    volumes:
+      - .:/kioku
+    working_dir: /kioku
 
+volumes:
+  stack-root:
```

Comment:
Oh, this `stack-root` volume definition is not used anywhere. It should be removed.

Reply:
I had based this setup on what I saw in telamon.

Reply:
We should go back and remove it there as well in that case.

Reply:
Already did and opened a PR for it :-)